packages/medusa/src/api/routes/store/carts/decorate-line-items-with-totals.ts

```diff
@@ -6,25 +6,34 @@ import { EntityManager } from "typeorm";
 export const decorateLineItemsWithTotals = async (
   cart: Cart,
   req: Request,
-  options: { force_taxes: boolean } = { force_taxes: false }
+  options: { force_taxes: boolean, transactionManager?: EntityManager } = { force_taxes: false }
```

Comment:
praise: Neat pattern!

Reply:
:) I think that at some point (after discussing the available options) this is something that could be used to replace the withTransaction